frontend/scripts/docs.js

```diff
@@ -13,7 +15,23 @@ app.use(
 ); // limit 1000 requests per 15 minutes
 
 app.use(express.static(path.join(__dirname, '../docs/build')));
+app.use(cors());
 
+app.use(
+  helmet({
+    contentSecurityPolicy: {
+      directives: {
+        baseUri: ["'none'"],
+        defaultSrc: ["'self'"],
+        frameAncestors: ["'none'"],
+        objectSrc: ["'none'"],
+        scriptSrc: ["'none'"]
+      }
+    },
```

Comment:
@matthew-grayson how do you plan to deploy this server? Another option is to use Cloudfront + S3, and you can customize the right headers from the Cloudfront distribution. That way you don't have to manage servers directly.

Reply:
CloudFront isn't available in GovCloud, and so the frontend is now deployed with serverless (see lz-infra branch). My intent is to attach the docs page to that frontend deployment.

Reply:
Ah I see! Makes sense, thanks.

Would the frontend be a different domain or just a path (like `crossfeed.cyber.dhs.gov/docs`?

Reply:
It's just like your example. The docs site will build out under the frontend like `crossfeed.cyber.dhs.gov/docs`, `crossfeed.cyber.dhs.gov/docs/api-reference`, etc.